iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -146,7 +145,6 @@ public static Collection inputs() throws Exception
     protected static RegistryManager registryManager;
     protected InternalClient internalClient;
     protected static RawTwinQuery scRawTwinQueryClient;
-    protected static DeviceTwin sCDeviceTwin;
```

Comment:
In order for some tests to have the twin service client use a proxy, and others to not, each test must have its own